src/mui/list/List.js

```diff
@@ -64,7 +67,7 @@ const styles = {
 export class List extends Component {
     constructor(props) {
         super(props);
-        this.state = { key: 0 };
+        this.state = {};
```

Comment:
Filters and action are using this state. Maybe we should refactor this case.

Here: https://github.com/marmelab/admin-on-rest/blob/next/src/mui/list/List.js#L183
And here: https://github.com/marmelab/admin-on-rest/blob/next/src/mui/list/List.js#L192